rotkehlchen/chain/substrate/manager.py

```diff
@@ -541,3 +593,20 @@ def get_last_block(
         trying with all the available nodes.
         """
         return self._get_last_block(node_interface)
+
+    def set_rpc_endpoint(self, endpoint: str) -> Tuple[bool, str]:
+        """Attempt to set the RPC endpoint for the user's own node.
+        If connection at endpoint is successful it will set `own_rpc_endpoint`.
+        """
+        if endpoint == '':
+            current_own_rpc_endpoint = getattr(self, 'own_rpc_endpoint', None)
```

Comment:
Why do this? Just set `self.own_rpc_endpoint = None` in the constructor. No need for `getattr`

Reply:
This is one of the things I addressed in the other feature I've been working. I will include it here.

Reply:
yeah include it and when we merge this, your other work can be rebased on this